# HG changeset patch
# User John Donoghue
# Date 1529599331 14400
#      Thu Jun 21 12:42:11 2018 -0400
# Node ID b3c68525ed82260891bc135ff546569e8d5a35ca
# Parent  9feda91c85927133b2c0b165a7d0e41291bcc9df
link oct files with liboctave and libinterp in cross compile tools (Bug #53627)

* src/mkoctfile.in.cc: us defined(CROSS)  as additional check for including libs/flags

diff --git a/src/mkoctfile.in.cc b/src/mkoctfile.in.cc
--- a/src/mkoctfile.in.cc
+++ b/src/mkoctfile.in.cc
@@ -159,17 +159,17 @@ initialize (void)
       + " -I" + quote_path (vars["OCTINCLUDEDIR"]);
 #endif
 
   if (vars["INCLUDEDIR"] != "/usr/include")
     DEFAULT_INCFLAGS += " -I" + quote_path (vars["INCLUDEDIR"]);
 
   std::string DEFAULT_LFLAGS;
 
-#if defined (OCTAVE_USE_WINDOWS_API)
+#if defined (OCTAVE_USE_WINDOWS_API) || defined(CROSS)
 
   // We'll be linking with -loctinterp and -loctave, so we need to know
   // where to find them.
 
   DEFAULT_LFLAGS += "-L" + quote_path (vars["OCTLIBDIR"]);
 #endif
 
   if (vars["LIBDIR"] != "/usr/lib")
@@ -872,17 +872,17 @@ main (int argc, char **argv)
                     << std::endl;
           return 1;
         }
     }
 
   if (link && ! objfiles.empty ())
     {
       std::string octave_libs;
-#if defined (OCTAVE_USE_WINDOWS_API)
+#if defined (OCTAVE_USE_WINDOWS_API) || defined(CROSS)
       octave_libs = "-loctinterp -loctave";
 #endif
 
       if (link_stand_alone)
         {
           if (! vars["LD_CXX"].empty ())
             {
               std::string cmd
